swift-ui-base/Extensions/ColorExtension.swift

```diff
@@ -7,27 +7,12 @@
 //
 
 import SwiftUI
-import Foundation
 
 extension Color {
   
-  static let darkGray = UIColor(red: 54, green: 61, blue: 79).asColor()
-  static let athensGray = UIColor(red: 229, green: 231, blue: 237).asColor()
-  static let altoGray = UIColor(red: 216, green: 216, blue: 216).asColor()
-  static let cadetBlue = UIColor(red: 167, green: 175, blue: 199).asColor()
-  static let lightGray = UIColor(red: 167, green: 175, blue: 199).asColor()
-  static let errorRed = UIColor(red: 255, green: 81, blue: 82).asColor()
-}
-
-extension UIColor {
-  convenience init(red: Int, green: Int, blue: Int) {
-    self.init(red: min(CGFloat(red), 255.0) / 255.0,
-              green: min(CGFloat(green), 255.0) / 255.0,
-              blue: min(CGFloat(blue), 255.0) / 255.0,
-              alpha: 1.0)
-  }
```

Comment:
Why removing this handy initializer?

Reply:
The idea is to create all as a color asset, that means that you can pass the `0.34234` RGB value directly to the asset. the initializer should not be used again.

Reply:
You can select `Floating point` there and that's it, only once to create the asset and you're ready to go. And actually the default selection on that dropdown menu is that one

<img width="259" alt="Screen Shot 2020-07-30 at 5 38 00 PM" src="https://user-images.githubusercontent.com/1467868/88972136-75640e00-d28b-11ea-9264-d715743ac80d.png">

Reply:
Also, `Color(red: , green: , blue: )` for values `0-255` already exists as part of the swiftUI core